maintainers/team-list.nix

```
@@ -638,15 +638,15 @@ with lib.maintainers; {
     enableFeatureFreezePing = true;
   };
 
-  nixos-modules = {
+  module-system = {
     members = [
       ericson2314
       infinisil
       qyliss
       roberth
     ];
-    scope = "Maintain nixpkgs module system internals.";
-    shortName = "NixOS Modules / internals";
+    scope = "Maintain Nixpkgs module system internals.";
+    shortName = "Module system / internals";
```

```suggestion
    scope = "Maintain the Nixpkgs module system and its use in NixOS. Out of scope are NixOS modules themselves.";
    shortName = "Module system";
```

---

By saying "out of scope are NixOS module themselves" you've basically negated "and its use in NixOS".
Some care for callers is implied by virtue of being a library, so I don't think we need to try to describe that here, in a roundabout way.

---

```suggestion
    scope = "Maintain the Nixpkgs module system.";
    shortName = "Module system";
```

Fair enough. I think the "internal" should be removed then though, because we also maintain its surface.